src/mlpack/tests/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # mlpack test executable.
 add_executable(mlpack_test
+  EXCLUDE_FROM_ALL
```

Comment:
Maybe this one causing the tests to stop running on CI?

Reply:
Yes, I need to update the CI configuration... I'll work through the failing builds to figure that out.